slice/Glacier2/Router.ice

```diff
@@ -31,6 +31,7 @@ module Glacier2
     }
 
     /// The Glacier2 specialization of the <code>Ice::Router</code> interface.
+    ["cs:attribute:System.ComponentModel.EditorBrowsable(System.ComponentModel.EditorBrowsableState.Never)"]
```

Comment:
cs:attribute has no effect on the generated Prx and PrxHelper. I assume this is on purpose!

Reply:
It seems like it is. And I think the behavior makes sense.
Even if they're generated code, proxies aren't really something a user would want to modify with attributes I think.